src/frontend/utils/theme/theme.ts

```diff
@@ -192,6 +192,9 @@ export const theme: ThemeType = {
     },
   },
   formField: {
+    checkBox: {
+      pad: {},
```

Comment:
This not bug but a modification made on purpose by grommet. You can modify your commit message and mentin in it this PR https://github.com/grommet/grommet/pull/6300/

Reply:
They have fix the issue https://github.com/grommet/grommet/commit/d6bac0cfdb4d1f36f4553b26bfc8f40d6c526761

Reply:
Nice, we just have to wait for a release.